ci/ci.sh

```diff
@@ -0,0 +1,128 @@
+#!/bin/bash
+set -e
+CI_DIR="/runner/_work/userspace-cni-network-plugin/userspace-cni-network-plugin/ci/"
+
+install_go_kubectl_kind(){
+wget -qO- https://golang.org/dl/go1.20.1.linux-amd64.tar.gz |tar -C /home/runner -xz
+export PATH="${PATH}:/home/runner/go/bin"
+echo "export PATH=\"${PATH}:/home/runner/go/bin/:home/runner/.local/bin/\"" >>~/.bashrc
+go install sigs.k8s.io/kind@v0.20.0
+
+wget -q https://dl.k8s.io/release/v1.27.3/bin/linux/amd64/kubectl -O /home/runner/go/bin/kubectl 
+chmod +x /home/runner/go/bin/kubectl
+}
+
+
+create_kind_cluster(){
+kind create cluster
+kubectl get all --all-namespaces
+
+#docker run -itd --device=/dev/hugepages:/dev/hugepages --privileged -v "$(pwd)/docker/vpp-centos-userspace-cni/:/etc/vpp/" --name vpp ligato/vpp-base
+sleep 10
+cd /runner/_work/userspace-cni-network-plugin/userspace-cni-network-plugin/
+
+docker build . -f ./docker/userspacecni/Dockerfile -t userspacecni:latest
+kind load docker-image userspacecni
+docker exec -i kind-control-plane bash -c "mkdir -p '/runner/_work/userspace-cni-network-plugin/'"
+docker cp "/runner/_work/userspace-cni-network-plugin/userspace-cni-network-plugin/" "kind-control-plane:/runner/_work/userspace-cni-network-plugin/"
+}
+
+deploy_multus(){
+## Multus main branch has major bugs so we fix version
```

Comment:
the issue with the conf file being created is now fixed on main branch

Reply:
It's still better to have a fixed version, as we don't want a repeat of what happened last time.
It took them 2 months to fix the issue